configure

```diff
@@ -1931,6 +1943,17 @@ else
 $as_echo "$as_me: building against PostgreSQL $version_num" >&6;}
 fi;
 
+# Define citus version related variables to use version checks
+citus_binary_version=$(grep "default_version" src/backend/distributed/citus.control | cut -d\' -f2)
```

Comment:
For safety, it's probably best to use `grep "^default version"` or something with an anchor like that.